test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -293,6 +293,7 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             Assert.True(context.HttpContext.Request.Body.CanRead);
         }
 
+        [ReplaceCulture("en-US", "en-US")]
```

Comment:
Don't use `currentCulture: "en-US"` in tests because it's indistinguishable from `InvariantCulture`. Stick with `[ReplaceCulture]` (w/o constructor parameters) everywhere.